cmd/cloud_login.go

```diff
@@ -68,6 +76,9 @@ the "k6 run -o cloud" command.
 //
 //nolint:funlen
 func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
+	warnColor := getColor(c.globalState.Flags.NoColor || !c.globalState.Stdout.IsTTY, color.FgRed)
```

Comment:
```suggestion
	warnColor := getColor(c.globalState.Flags.NoColor || !c.globalState.Stdout.IsTTY, color.FgYellow)
```
Aren't warnings generally yellow/orange? :thinking:

Reply:
I'm fine with any, what do you suggest @olegbespalov @oleiade? 
Let's go with `color.FgYellow` as suggested by @codebien?

Reply:
Indeed 👍🏻